packages/devtools_app/lib/src/ui/search.dart

```diff
@@ -454,3 +457,14 @@ class SearchNavigationControls extends StatelessWidget {
     );
   }
 }
+
+mixin SearchableDataMixin {
+  bool isSearchMatch = false;
+  bool isActiveSearchMatch = false;
+  String get tooltip => '';
```

Comment:
Why does this mixin have a tooltip? That seems unrelated to the rest of what the mixin is doing.
Also consider renaming this 
SearchStateMixin or similar as this mixin seems to be the state of whether something is a search match. That name suggestion isn't great so feel free to ignore or coome up with something better.

Reply:
I did this to avoid having to add another Mixin, but you're right it doesn't really fit here. I was hoping to be able to reuse this tooltip in the table code in a follow up PR, but for now I created a mixin `FlameChartDataMixin` that has the tooltip getter, and added that to the data models we use in flame charts. I can generalize in the follow up PR where I address the table search perf.